site/docs/v1/tech/tutorials/integrate-laravel-api.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: doc
+title: Integrate Your Laravel API With FusionAuth
+description: Integrate your Laravel API with FusionAuth
+navcategory: getting-started
+prerequisites: Composer, Docker and at least PHP 8.2
+technology: Laravel
+language: PHP
+---
+
+## Integrate Your {{page.technology}} API With FusionAuth
+
+{% include docs/integration/_intro-api.md %}
+
+## Prerequisites
+
+{% include docs/integration/_prerequisites.md %}
+
+## Download and Install FusionAuth
+
+{% include docs/integration/_install-fusionauth.md %}
+
+## Create a User and an API Key
+
+{% include docs/integration/_add-user.md language=page.language %}
+
+## Configure FusionAuth
+
+Next, you need to set up FusionAuth. This can be done in different ways, but we’re going to use the [{{page.language}} client library](/docs/v1/tech/client-libraries/php). You can use the client library with an IDE of your preference as well.
+
+First, make a directory:
+
+```shell
+mkdir setup-fusionauth && cd setup-fusionauth
+```
+
+Now, copy and paste the following file into `composer.json`.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/php/composer.json %}
+```
+
+Install the dependencies.
+
+```shell
+composer install
+```
+
+Then copy and paste the following file into `setup.php`.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/php/setup.php %}
+```
+
+Then, you can run the setup script.
+
+{% include _callout-note.liquid content="The setup script is designed to run on a newly installed FusionAuth instance with only one user and no tenants other than `Default`. To follow this guide on a FusionAuth instance that does not meet these criteria, you may need to modify the above script. <br><br> Refer to the [PHP client library](/docs/v1/tech/client-libraries/php) documentation for more information." %}
+
+This will create the FusionAuth configuration for your {{page.technology}} API.
+
+```shell
+php setup.php "YOUR_API_KEY_FROM_ABOVE"
+```
+
+If you want, you can [log into your instance](http://localhost:9011) and examine the new Application the script created for you.
+
+## Create Your {{page.technology}} API
+
+Now you are going to create a {{page.technology}} application. While this section uses a simple {{page.technology}} application, you can use the same steps to integrate any {{page.technology}} application with FusionAuth.
+
+First, make a directory:
+
+```shell
+mkdir ../setup-laravel && cd ../setup-laravel
+```
+
+Next, create a simple {{page.technology}} template using Laravel's build script.
+
+```shell
+curl -s "https://laravel.build/fusionauth-example-laravel-api?with=mariadb" | bash
```

Comment:
Why with mariadb?

Reply:
By default, the [Laravel build script](https://laravel.build/fusionauth-example-laravel-api) would install `mysql redis meilisearch mailpit selenium`, which is too much. So we'd only want a database there. MariaDB is generally preferred over MySQL due to its entire free+open-source approach (MySQL only offers some resources in Enterprise versions). I added this note there.